### PR TITLE
Fix/thanks

### LINKED ISSUE #25: Replace star with brownie or other desert emoji when showing about
https://github.com/FreeCodeCamp/camperbot/blob/master/nap/lib/bot/cmds/thanks.js#L139

### LINKED ISSUE #49: Specifically exclude the word `tax` from the "Thank You" Regex
Because this is just silly:
image

### LINKED ISSUE #76: Camperbot to ignore `@/all` thanks

### LINKED ISSUE #86: Improve the 'about' command
I've got an idea: camperbot will use :star2: `:star2:` for `1000 brownies` and above.

Add this whenever you want, this feature isn't necassary. But I still want it to be added.

### PR DESCRIPTION
**Pull request by noncentz**
Ignore thanks in usernames
Ignore tax and tanks triggering thanks command. Fixes #49 or https://github.com/dcsan/gitterbot/issues/142
Removes missing thanks 'message' command
Removes botx command
Replace star emoji with cookie. Fixes #25
Correct typo in readme 'campberbot'
Fix about command. Fixes #86 - 'superstar' status.

Adds love command such as  `show me some love camperbot` or `we luv you cbot`
https://regex101.com/r/pT0zJ1/3

### CODE REVIEW

**Comment by SaintPeter** [comment]
@abhisekp Would you be able to spin this up and test it?  I think you may have the most experience.

@noncentz Thanks!  Really appreciate you resubmitting!  Your code looks solid to me.

File: lib/bot/cmds/thanks.js [hunk 1]
```diff
@@ -35,7 +35,8 @@ const thanksCommands = {
 
     const namesList = mentions.reduce((userList, mention) => {
       const toUser = mention.screenName.toLowerCase();
-      if (toUser !== fromUser && userList.indexOf(toUser) === -1) {
+
+      if (toUser !== fromUser && toUser !== 'all'  && userList.indexOf(toUser) === -1) {
```
Comment (abhisekp) [inline lib/bot/cmds/thanks.js]: I've addressed this in https://github.com/FreeCodeCamp/camperbot/pull/83

Comment (abhisekp) [inline lib/bot/cmds/thanks.js]: @noncentz can you please remove the edit to this line so that this PR can be merged happily ever after?

**Comment by abhisekp** [comment]
This PR also fixes https://github.com/dcsan/gitterbot/issues/142

File: lib/bot/cmds/thanks.js (hunk 1 above)
Comment (noncentz) [inline lib/bot/cmds/thanks.js]: @abhisekp , is it good now? https://github.com/FreeCodeCamp/camperbot/pull/78/commits/2e431ed2f3da662ae38c5dd690376b09450897aa

File: lib/bot/cmds/thanks.js [hunk 2]
```diff
@@ -6,15 +6,15 @@ const Utils = require('../../../lib/utils/Utils'),
...
-  messages: {
-    wikiHint: function(fromUser) {
-      const wikiUrl = '(https://github.com/freecodecamp/' +
-                    'freecodecamp/wiki/wiki-style-guide)';
-
-      return '\n> hey @' + fromUser + ' if you found this info helpful ' +
-        ':point_right: *[consider adding a wiki article!]' + wikiUrl + '*';
-    }
-  },
+  // messages: {
```
Comment (abhisekp) [inline lib/bot/cmds/thanks.js]: **Pending:** Test this in bonfire rooms

File: data/rooms/RoomMessages.js [hunk 3]
```diff
@@ -50,8 +44,17 @@ const AllRoomMessages = [
     chance: 1
   },
   {
-    regex: /\bth?a?n?[xk]s?q?\b/gim,
+    //tests: https://regex101.com/r/hH5cN7/42
+    regex: /(?:^|\s)(?:(?:th(?:n[qx]|x)|t[xyq])|than[kx](?:[sxz]){0,2}(?:[uq]|y(?:ou)?)?)\b/i,
```
Comment (abhisekp) [inline data/rooms/RoomMessages.js]: This regex seems perfect. :+1: 

Pending Test

File: data/rooms/RoomMessages.js [hunk 4]
```diff
@@ -50,8 +44,17 @@ const AllRoomMessages = [
...
     func: BotCommands.thanks
+  },
+  {
+    //tests: https://regex101.com/r/pT0zJ1/2
+    regex: /(?:[^@`]|\s+|^)(?:love|luv)\s?(?:u|you|me)?,?\s?(?:cbot|@?camperbot)\b/i,
```
Comment (abhisekp) [inline data/rooms/RoomMessages.js]: Document this new command clearly in the [**PR message**](https://github.com/FreeCodeCamp/camperbot/pull/78#issue-148545636).

**Comment by abhisekp** [comment]
Write a detailed changelog of the changes made in this PR in the [**PR Message**](https://github.com/FreeCodeCamp/camperbot/pull/78#issue-148545636)

File: lib/bot/cmds/thanks.js [hunk 5]
```diff
@@ -106,7 +106,7 @@ const thanksCommands = {
       const username = blob.response.about.username,
             about = blob.response.about,
             uri = 'http://www.freecodecamp.com/' + username;
-      str = '> :star: ' + about.browniePoints + ' | @' + username + ' | ';
+      str = '> :cookie: ' + about.browniePoints + ' | @' + username + ' | ';
```
Comment (abhisekp) [inline lib/bot/cmds/thanks.js]: @noncentz can you implement this https://github.com/FreeCodeCamp/camperbot/issues/86 here? (_optional_)

File: data/rooms/RoomMessages.js [hunk 6]
```diff
@@ -50,8 +44,17 @@ const AllRoomMessages = [
...
+    func: function(input) {
```
Comment (abhisekp) [inline data/rooms/RoomMessages.js]: @noncentz write a documentation in a comment about the command i.e. what it does and when it does, etc. 
E.g.

```js
/**
* This command loves camperbot.
**/
```

**Comment by ltegman** [comment]
LGTM

**Pull request merged**